fhir-api/authentication.py

```
@@ -0,0 +1,28 @@
+from flask import request
+from functools import wraps
+import jwt
+import os
+
+from errors import AuthenticationError
+
+JWT_PUBLIC_KEY = os.getenv("JWT_PUBLIC_KEY")
```

maybe we should raise at application startup if this is missing